srcpkgs/zutty/template

```diff
@@ -1,17 +1,17 @@
 # Template file for 'zutty'
 pkgname=zutty
-version=0.11
+version=0.12
 revision=1
 build_style=waf3
-hostmakedepends="pkg-config"
+hostmakedepends=pkg-config
 makedepends="python3-devel libglvnd-devel MesaLib-devel freetype-devel libXmu-devel"
-depends="font-misc-misc"
+depends=font-misc-misc
 short_desc="X terminal emulator rendering through OpenGL ES Compute Shaders"
 maintainer="Javier Caballero <jacallo@protonmail.com>"
-license="GPL-3.0-or-later"
-homepage="https://tomscii.sig7.se/zutty"
+license=GPL-3.0-or-later
+homepage=https://tomscii.sig7.se/zutty
 distfiles="https://github.com/tomszilagyi/zutty/archive/refs/tags/${version}.tar.gz"
-checksum=3e77abb424f2f6044686fd3a3a22793e537bf5e403fe2b6ca8543daf4f3186f9
+checksum=750daed8625a2486a33b872d59242efba7ee21af282ccfd4a7dbf12d21d7cee7
 
 post_patch() {
 	vsed -i wscript -e "s|, '-march=native', '-mtune=native'||g"
```

Comment:
if you look at the CI output, you will see that this line doesn't change anything and causes the build to fail